apps/remote_control/lib/lexical/remote_control/code_intelligence/ast.ex

```diff
@@ -0,0 +1,101 @@
+defmodule Lexical.RemoteControl.CodeIntelligence.Ast do
```

Comment:
I think in the near future, `find references` and `go to definition` will use the functions in this module too, so I put them here.